libraries/AP_ADSB/AP_ADSB.cpp

```diff
@@ -50,12 +51,12 @@ AP_ADSB *AP_ADSB::_singleton;
 
 // table of user settable parameters
 const AP_Param::GroupInfo AP_ADSB::var_info[] = {
-    // @Param: ENABLE
-    // @DisplayName: Enable ADSB
-    // @Description: Enable ADS-B
-    // @Values: 0:Disabled,1:Enabled
+    // @Param: TYPE
+    // @DisplayName: ADSB Type
+    // @Description: Type of ADS-B hardware for ADSB-in and ADSB-out configuration and operation. If any type is selected then MAVLink based ADSB-in messages will always be enabled
+    // @Values: 0:Disabled,1:uAvionix-MAVLink,2:Sagetech
     // @User: Standard
-    AP_GROUPINFO_FLAGS("ENABLE",     0, AP_ADSB, _enabled,    0, AP_PARAM_FLAG_ENABLE),
+    AP_GROUPINFO_FLAGS("TYPE",     0, AP_ADSB, _type[0],    0, AP_PARAM_FLAG_ENABLE),
```

Comment:
This name change will need a param conversion will it not?

Reply:
I thought so too but I don't think it's needed since it's the same index, same type, and serves the same purpose. However, we'll need to update the wiki. I confirmed this with @peterbarker verbally but I'd love for him to take a look at that specific thing in this PR